staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/strategy.go

```diff
@@ -188,8 +189,19 @@ func (a customResourceStrategy) Validate(ctx context.Context, obj runtime.Object
 }
 
 // WarningsOnCreate returns warnings for the creation of the given object.
-func (customResourceStrategy) WarningsOnCreate(ctx context.Context, obj runtime.Object) []string {
-	return nil
+func (a customResourceStrategy) WarningsOnCreate(ctx context.Context, obj runtime.Object) []string {
```

Comment:
make a single function that computes warnings for `obj, old runtime.Object`, and call it with `obj, nil` from WarningsOnCreate